.github/workflows/iroha2-custom-image.yml

```diff
@@ -60,7 +65,7 @@ jobs:
           retention-days: 1
 
   registry-profiling-image:
-    if: ${{ inputs.BUILD_GLIBC_IMAGE == 'false' }} and ${{ inputs.BUILD_ALPINE_IMAGE == 'false' }}
+    if: ${{ inputs.BUILD_PROFILER_IMAGE == 'true' }} and ${{ inputs.BUILD_GLIBC_IMAGE == 'false' }} and ${{ inputs.BUILD_ALPINE_IMAGE == 'false' }}
```

Comment:
To avoid the invalid 5 = 2^3 - 3 input patterns, I think we only have to check `inputs.PROFILER_NAME`, which should be either profiling, glibc, or alpine

Reply:
Or, check one of the three flags to build and push images in parallel in a single workflow run

Reply:
Checkbox [hasn't implemented](https://github.com/github/vscode-github-actions/issues/190) in GHA yet (at least, officially)